Modernize Custom Property Editors/Delivery API away from IPublishedSnapshotAccessor (#7573)
umbraco/UmbracoDocs: 16/umbraco-cms/reference/content-delivery-api/custom-property-editors-support.md

    private DeliveryApiCustomPicker? BuildDeliveryApiCustomPicker(object inter, bool expanding)
    {
        if (!Guid.TryParse(inter as string, out Guid id))
        {
            return null;
        }
        
        var content = publishedContentCache.GetById(id);

        if (content is null)
        {
            return null;
        }
        
        return new DeliveryApiCustomPicker { Id = id };
    }
}
```
{% endcode %}

{% hint style="info" %}

> Umbraco developers will need to inject `IPublishedContentCache`, `IPublishedMediaCache`, `IPublishedMemberCache` and `IPublishedContentTypeCache` dependencies individually, instead of injecting the `IPublishedSnapshotAccessor` as would have been done previously.

Reviewer: ```suggestion
Umbraco developers need to inject `IPublishedContentCache`, `IPublishedMediaCache`, `IPublishedMemberCache`, and `IPublishedContentTypeCache` individually, instead of injecting the `IPublishedSnapshotAccessor` as in previous versions.
```
